src/components/DashboardResultIndicatorsSection.vue

```diff
@@ -144,14 +144,15 @@ export default {
 
   data: () => ({
     activeTab: 0,
+    activeResultIndicator: null,
+    graph: null,
     downloadOption: '',
     progressCollection: [],
     unexpiredGoals: [],
-    latestProgressRecord: 0,
     resultIndicatorPeriods: Object.values(RESULT_INDICATOR_PERIODS).map(
       (period) => period
     ),
-    currentResultIndicatorPeriod: RESULT_INDICATOR_PERIODS.sixmonths,
+    currentResultIndicatorPeriod: RESULT_INDICATOR_PERIODS.all,
```

Comment:
Uten å ha sett på resten av kodeendringene her, så lurer jeg nesten på om dette forutsetter at vi får på plass en loading indicator og/eller får den uthentingen til å gå raskere, for med prod-dataene kjennes det på grensen til buggy ut. :/

(Men denne endringen er vel urelatert til resten? A.K.A. kunne gått i en egen PR.)

Reply:
Ja, for så vidt enig i begge deler. Reverterer, og ser på muligheten for å gjøre denne endringen + TODO-en over sammen og separat.